vpr/src/place/place_util.cpp

```diff
@@ -328,3 +328,56 @@ double get_std_dev(int n, double sum_x_squared, double av_x) {
     /* Very small variances sometimes round negative. */
     return (std_dev > 0.) ? sqrt(std_dev) : 0.;
 }
+
+/**
+ * @brief Builds (alloc and load) legal_pos that holds all the legal locations for placement
+ *
+ *   @param legal_pos
+ *              a lookup of all subtiles by sub_tile type
+ *              legal_pos[0..device_ctx.num_block_types-1][0..num_sub_tiles - 1][0..num_legal - 1] = t_pl_loc for a single
```

Comment:
I'd get rid of num_legal in this comment and use size-1 instead (or just say you get a vector of all the legal locations of the proper tile type and sub_tile type and remove the third [0..num_legal-1] dimension in the comment.).

Reply:
resolved!